src/styles/cart.css

```diff
@@ -0,0 +1,61 @@
+.cart-container{
+    padding: 5rem 0;
+    height: 100vh;
+    /* background-color: var(--lynx-white-color); */
+}
+
+.cart{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: center;
+    align-items: center;
+}
+
+.bg-color{
+  background-color: var(--white-color);
+}
+
+.quantity{
+    display: flex;
+    gap: 1.25rem;
+    align-items: center;
+}
+
+.plus-minus{
+    padding: 0.25rem 0.5rem;
+    cursor: pointer;
+}
+
+.qty-input{
+    width: 5rem;
+    border: transparent;
+    outline: none;
+    text-align: center;
+    padding: 0.25rem 0.5rem;
+    border-top: 1px solid var(--lynx-grey-color);
+    border-bottom: 1px solid var(--lynx-grey-color);
+}
+
+.cart-btn{
+    width: 95%;
+}
+
+.billing-details{
+    width: 18%;
+    padding: 2rem;
+    background-color: var(--white-color);
+    /* display: flex; */

```

Comment:
remove if not needed

Reply:
Done [346210e](https://github.com/Manpreet-hub/ecommerce-react/pull/4/commits/346210efcfbdd93bcf04432da991ab1028dd18a2)